debian/postinst

```diff
@@ -27,10 +28,30 @@ case "$1" in
         ln -s /etc/nginx/sites-available/default /etc/nginx/sites-enabled/default
         rm -f /etc/kolibri/nginx_default
     fi
-    echo "include $DAEMON_HOME/nginx.conf;" > /etc/kolibri/nginx.d/099-user.conf
+    echo "include $KOLIBRI_HOME/nginx.conf;" > /etc/kolibri/nginx.d/099-user.conf
+
+    # Write logrotate configuration if it's not already written
+    LOGROTATE_CONF="/etc/logrotate.d/kolibri"
+    if [ -e "/etc/logrotate.d" ] # && ! [ -e "$LOGROTATE_CONF" ]
+    then
+      # Issues with string substitution for "\n" caused this pattern
+      echo "\"${KOLIBRI_HOME}/logs/uwsgi.log\" {" > "$LOGROTATE_CONF"
+      echo "  copytruncate" >> "$LOGROTATE_CONF"
+      echo "  weekly" >> "$LOGROTATE_CONF"
+      echo "  rotate 150" >> "$LOGROTATE_CONF"
+      echo "  size 2M" >> "$LOGROTATE_CONF"
+      echo "  compress" >> "$LOGROTATE_CONF"
+      echo "  delaycompress" >> "$LOGROTATE_CONF"
+      echo "  missingok" >> "$LOGROTATE_CONF"
+      echo "  notifempty" >> "$LOGROTATE_CONF"
+      echo "  create 0644 $KOLIBRI_USER $KOLIBRI_GROUP" >> "$LOGROTATE_CONF"
```

Comment:
I didn't even know there is a $KOLIBRI_GROUP , why is it needed here?

Reply:
I understand from the `create` syntax that in order to set ownership for the rotated log files, you need to declare both user and group: `$KOLIBRI_GROUP` is defined further up as the primary group of `$KOLIBRI_USER`